lib/base-compiler.ts

```diff
@@ -880,7 +884,7 @@ export class BaseCompiler implements ICompiler {
         let libPaths: string[] = [];
         let staticLibLinks: string[] = [];
 
-        if (filters.binary) {
+        if (filters.binary || filters.binaryobject) {
```

Comment:
in what situations is this needed? The binaryobject things should be pre-link, right?

Reply:
You're right, that's me duplicating `binary` behavior as the base, but this is not needed

Reply:
done